packages/optimizer/spec/helpers/TransformerRunner.js

```diff
@@ -58,22 +58,28 @@ module.exports = function(testConfig) {
           input = input.substring(indexEndConfig + CONFIG_END_TOKEN.length);
         }
 
-        const tree = treeParser.parse(input);
+        const tree = await treeParser.parse(input);
 
         // parse expected output
         const expectedOutputPath =
           join(
               testDir,
             testConfig.validAmp ? 'expected_output.valid.html' : 'expected_output.html',
           );
-        const expectedOutput = getFileContents(expectedOutputPath);
+        let expectedOutput = '';
+        try {
+          expectedOutput = getFileContents(expectedOutputPath);
+        } catch (e) {
+          // no snapshot written yet
```

Comment:
nit: I don't really understand what this comment means. Maybe it's worth explaining more?

Reply:
done